src/Adapters/Storage/Mongo/MongoSchemaCollection.js

```diff
@@ -274,6 +275,21 @@ class MongoSchemaCollection {
         }
       });
   }
+
+  async updateFieldOptions(className: string, fieldName: string, fieldType: any) {
+    const fieldOptions = _.cloneDeep(fieldType);
```

Comment:
is `cloneDeep` needed here? could `{...fieldType}` do the same?

Reply:
@sadortun  seems to have replaced my code thats non eslint friendly
Here my original code:

```
// eslint-disable-next-line no-unused-vars
    const { type, targetClass, ...fieldOptions } = fieldType;
```

I'm not a fan of mutations so i'll push my original code again :)

Reply:
@Moumouls See previous discussion 

https://github.com/parse-community/parse-server/pull/7418/files/aa9b15e143b99fe14b58a4c56d5c8c01c4a97575#r687211652


@dblythy Yes! 

---

Please lmk what you want here :) I also prefer @Moumouls Syntax but I think @mtrezza  had some objection.